package.json

```diff
@@ -19,12 +26,14 @@
     "next": "13.1.6",
     "react": "18.2.0",
     "react-dom": "18.2.0",
+    "sass": "^1.58.3",
```

Comment:
Looking at the PR, do we still need this package (and node-sass)?

Reply:
probably not until snowfall. Or we try making a tailwind snowfall subrepo